explainer.md

```diff
@@ -96,6 +96,50 @@ become [much more expensive](https://pq-crystals.org/dilithium/). The
 flexibility will also simplify the post-quantum transition itself, as different
 post-quantum CAs may be added at different times.
 
+### Minimizing Server Operator Burden
+
+In reasoning through PKI deployment strategies, one of our primary goals is to
+minimize server operator burden. Where possible, permit server operators to make
+simpler, high-level decisions (e.g. which ACME endpoint(s) cover the clients
+they need), with automation handling the rest.
+
+This is important because there are many HTTPS servers on the web, operated by
+different parties. While some strategies, such as fingerprinting or manually
+collecting an optimal set of intermediates, can be done by a small number of
+large hosting providers, they are impractical for a more diverse set of HTTPS
+servers. Avoiding this overhead helps both support the current server diversity
+and to maintain it going forward.
+
+This means:
+
+* **Prefer fewer mechanisms with broad coverage, over more mechanisms that cover
+  individual scenarios.** While one cannot solve every problem at once, broad
+  coverage minimizes software updates needed. If one mechanism addresses
+  multiple similar PKI scenarios, it can be implemented in server software once,
+  then used whenever each of these scenarios arise. In contrast, inventing a
+  bespoke mechanism for each kind of PKI transition resets the protocol
+  deployment clock each time.
+
+* **Prefer unambiguous, systematic server decisions.** While an individual
+  deployment might observe that, e.g., all of their newer clients support some
+  CA, or use more complex fingerprinting mechanisms, this requires server
+  operators to make risk assessments about imprecise properties of their client
+  base. It cannot be generalized and automated. In contrast, a mechanism that
+  directly negotiates the selection critera can be implemented in the TLS
+  library, with HTTPS software and server operators only providing the inputs.
+
+* **Shift work to CAs and root programs.** CAs are well-positioned to react to
+  changes in root program requirements. They mint the certificates and
+  maintain relationships with both server operators and the root programs whose
+  requirements they follow. User security improvements are dramatically easier
+  to deploy when only CAs and root programs need to act.
+
+* **Minimize risky decisions during incident response.** Root program
+  requirements naturally change over time, such as responding to a compromised
+  CA. Failing to do this puts user security at risk. However, from the server
+  operator's perspective, a change to their certificate profile risks breaking
```

Comment:
What does certificate profile mean here? In PKI terms, this usually refers to the issuance template for a given certificate, but I think this is referring to changes in trust for a given certificate?